### DISCUSSION TITLE
"add a readme" prompt still displays for empty README.md files

### DISCUSSION THREAD

**Discussion by benrosen** [community/community#52625] [Repositories]
### Select Topic Area

Product Feedback

### Body

I created a new README.md file with nothing in it. I committed it and pushed it to my remote. Even though I have a README.md file, I still see the "add a readme" prompt. I suspect this is because my README.md file is empty.

This isn't really a problem, but it is an edge case I thought you folks might want to hear about. Feel free to close/disregard/whatever if this is not helpful :)

<img width="913" alt="Screen Shot 2023-04-12 at 9 05 28 AM" src="https://user-images.githubusercontent.com/1329235/231466957-bd0d13e7-e96f-4d7d-825d-cb525d6c9bc6.png">

**Comment by pefribeiro** [comment]
This looks like a bug, because if you do go on and do "Add a README" it is then unable to save it, reporting "There was an error committing your changes: File could not be edited." as seen below:

![Screenshot 2023-05-09 at 15 42 49](https://github.com/community/community/assets/5167368/d0dd71ce-658d-4102-89bb-6b5662d040e4)
